exporters/cmake/OpenSSLConfig.cmake.in

```diff
@@ -129,13 +129,15 @@ set(OPENSSL_VERSION_FIX "${OpenSSL_VERSION_PATCH}")
 set(OPENSSL_FOUND YES)
 
 # Directories and names
-set(OPENSSL_LIBRARY_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::LIBDIR_REL_PREFIX, 1); -}")
-set(OPENSSL_INCLUDE_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::INCLUDEDIR_REL_PREFIX, 1); -}")
-set(OPENSSL_ENGINES_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::LIBDIR_REL_PREFIX, 1); -}/{- unixify($OpenSSL::safe::installdata::ENGINESDIR_REL_LIBDIR, 1); -}")
-set(OPENSSL_MODULES_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::LIBDIR_REL_PREFIX, 1); -}/{- unixify($OpenSSL::safe::installdata::MODULESDIR_REL_LIBDIR, 1); -}")
-set(OPENSSL_RUNTIME_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::BINDIR_REL_PREFIX, 1); -}")
+set(OPENSSL_LIBRARY_DIR "${_ossl_prefix}/{- unixify($OpenSSL::safe::installdata::LIBDIR_REL_PREFIX[0], 1); -}")
+set(OPENSSL_INCLUDE_DIR{- $OUT = '';
+                          $OUT .= ' "${_ossl_prefix}/' . $_ . '"'
+                              foreach (@OpenSSL::safe::installdata::INCLUDEDIR_REL_PREFIX); -})
```

Comment:
I am probably confused here - does this assume two include directories for the target installation? Two include dirs are needed only during the build of the OpenSSL itself. Once it is installed there should be only single one for apps to use. But maybe I am just confused by this being in installdata??

Reply:
Yes you are.  There are two variants of OpenSSLConfig.cmake:

- one that ends up in $BLDSRC, i.e. the top of the build tree, which is used by people who want to link directly against that build without having to install.
  That's what the CMake `export()` function provides.
- one that ends up in $BLDSRC/exporters, which is the variant that's actually getting installed in the appropriate CMake install directory.
  That's what the CMAKE `install(EXPORT)` function provides.

The one in $BLDSRC needs two include dirs, one being $BLDSRC/include and one being $SRCDIR/include

Reply:
See the description of this PR 😉

Reply:
So maybe I should rename `installdata` to something else?  `exportdata`, perhaps?